core/src/main/java/io/grpc/PickFirstBalancerFactory.java

```diff
@@ -70,7 +72,7 @@ public void handleResolvedAddressGroups(
       EquivalentAddressGroup newEag = flattenEquivalentAddressGroup(servers);
       if (subchannel == null) {
         subchannel = helper.createSubchannel(newEag, Attributes.EMPTY);
-        helper.updatePicker(new Picker(PickResult.withSubchannel(subchannel)));
+        helper.updateBalancingState(CONNECTING, new Picker(PickResult.withSubchannel(subchannel)));
```

Comment:
The new subchannel won't try to connect until an RPC is started.
The initial state of a new subchannel is `IDLE`. This should pass `IDLE`.